WordPress/Classes/Services/PostServiceRemoteFactory.swift

```diff
@@ -16,4 +16,36 @@ import WordPressKit
             return nil
         }
     }
+
+    @objc func restRemoteFor(siteID: NSNumber, context: NSManagedObjectContext) -> PostServiceRemoteREST? {
+        guard let api = apiForRESTRequest(using: context) else {
+            return nil
+        }
+
+        return PostServiceRemoteREST(wordPressComRestApi: api, siteID: siteID)
+    }
+
+    // MARK: Private methods
+
+    /// Get the api to use for making REST requests.
+    ///
+    /// - Returns: an instance of WordPressComRestApi
+    private func apiForRESTRequest(using context: NSManagedObjectContext) -> WordPressComRestApi? {
+        let accountService = AccountService(managedObjectContext: context)
+
+        guard let account = accountService.defaultWordPressComAccount(),
+              let api = account.wordPressComRestApi else {
```

Comment:
This could just be one line:
`guard let api = accountService.defaultWordPressComAccount()?.wordPressComRestApi`

Reply:
Noted!